tests/renderGistCard.test.js

```diff
@@ -0,0 +1,79 @@
+import { renderGistCard } from "../src/cards/gist-card";
+import { describe, expect, it } from "@jest/globals";
```

Comment:
Maybe we can add a snapshot test for consistency with the other cards (see https://github.com/anuraghazra/github-readme-stats/blob/master/tests/fetchTopLanguages.test.js for an example) 🤔?

Reply:
I planned to cover new gist card code with tests completely. As you can see i have some todos at the bottom of pull request description and writing tests still actual. Currently i wrote all basic tests including rendering, fetching data and e2e. They works. We can merge this pull request now and increase tests converage to 100% step by step later. Or we can wait for full converage. On your chice, i have converted this pull requests from draft into ready to merge. Feel free to merge.